client/store/modules/revisions.js

```diff
@@ -40,6 +42,14 @@ action(function fetch() {
   });
 });
 
+action(function restore(revision) {
+  const { id, courseId } = revision.state;
+  const url = `/courses/${courseId}/revisions/${id}/restore`;
+  this.commit('update', { ...revision, restored: true });
+  return request.post(url, { revision })
+    .then(({ data: { data } }) => this.commit('add', data));
```

Comment:
Instead double destructuring of data there can be nice little helper😀
```js
const extractData = res => res.data.data;
```

Reply:
I left it as it was to be more consistent like in other store modules. Maybe refactor this in every module in new PR? :bowtie:

Reply:
Let's leave it for next PR :hourglass: